### PR TITLE
Adding/removing public keys

### PR DESCRIPTION
**Pull request by organman91**
@mpdavis please review

### CODE REVIEW

File: server/manage.py [hunk 1]
```diff
@@ -102,6 +102,30 @@ def get_keys():
 
 
 @manager.command
+def add_public_key(username, filename):
+    from scanner.api import ssh_key
+    host = 'vinz-ubuntu.student.iastate.edu'
+    with open(filename) as f:
+        result = ssh_key.add_user_public_key(username, [host], f.read())
+    if result[host]:
+        print 'Success!'
```
Comment (mpdavis) [inline server/manage.py]: I would make these give more info.  Something like:

`'SSH Key successfully added for %s' % username`

File: server/scanner/api/ssh_key.py [hunk 2]
```diff
@@ -59,19 +59,53 @@ def check_user_ssh_key(username, inventory, vinz_private_key_path):
...
+    :return: A dict of whether operation succeeded on each host
     """
+    module_args = 'state=present user=%s key=\"%s\"' % (username, user_public_key)
+    runner = VinzRunner(hosts=hosts, module_name='authorized_key', module_args=module_args)
+    response = runner.run()
+    results = {}
+    for host in hosts:
+        if not host in response['contacted']:
+            #host is dark
+            results[host] = False
```
Comment (mpdavis) [inline server/scanner/api/ssh_key.py]: I said before that this should just return True/False, but I think we should change that.  I will want error messages to log in the event that something goes wrong here. 

I would say return something like this:

``` python
results = {
    'host1': {
        'success' : True,
    },
    'host2': {
        'success': False,
        'error': 'Unable to contact server host2'
    }
]
```

File: server/manage.py [hunk 3]
```diff
@@ -102,6 +102,30 @@ def get_keys():
...
+    else:
+        print 'Fail'
```
Comment (mpdavis) [inline server/manage.py]: I would print the actual error message here.

**Comment by organman91** [comment]
That commit should do it.

**Comment by mpdavis** [comment]
Looks good.

**Pull request merged**